src/adapter/src/catalog/config.rs

```diff
@@ -130,6 +136,7 @@ impl Default for ClusterReplicaSizeMap {
                     cpu_limit: None,
                     scale,
                     workers: scale.into(),
+                    credits_per_hour: (scale * scale).into(),
```

Comment:
This should be `scale` only I think.

Reply:
I know nothing about how credits per hour are calculated, what's the rationale of just using `scale` instead of `scale * worker`?

Reply:
This entire code here is only used for Mz instances that are used for testing purposes. I would say that all workers that run on the same machine should count as a single credit. This makes reasoning about tests easier I think. But I do not really have a very strong opinion on this.

Reply:
If it makes testing easier, then I think it's worth it. I just pushed the changes.